packages/forms/src/components/UniversalPickerOption/index.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useUniqueIdentifier } from '@rainbow-modules/hooks';
+import DefaultItem from './defaultItem';
+import { StyledContainer, StyledInput, StyledLabel } from './styled';
+import { Consumer as UniversalPickerConsumer } from '../UniversalPicker/context';
+
+const PickerOption = (props) => {
+    const { id, style, className, onChange, ariaDescribedby, component, ...rest } = props;
+    const { name, disabled, multiple, groupName, value } = rest;
+
+    const [isFocused, setFocused] = useState(false);
+    const [isHover, setHover] = useState(false);
+
+    const type = multiple ? 'checkbox' : 'radio';
+
+    const inputId = useUniqueIdentifier('universal-picker_item');
+
+    const isSelected = () => {
+        if (multiple) {
+            return Array.isArray(value) && value.includes(name);
+        }
+
+        return typeof value === 'string' && name === value;
+    };
+
+    const state = { isSelected: isSelected(), isFocused, isHover };
+
+    const Item = component || DefaultItem;
+
+    return (
+        <StyledContainer className={className} style={style} id={id}>
+            <StyledInput
+                as="input"
+                type={type}
+                id={inputId}
+                name={groupName}
+                checked={isSelected()}
+                aria-describedby={ariaDescribedby}
+                onChange={(event) => onChange(name, event.target.checked)}
+                disabled={disabled}
+                onFocus={() => setFocused(true)}
+                onBlur={() => setFocused(false)}
+            />
+
+            <StyledLabel
+                htmlFor={inputId}
+                onMouseEnter={() => setHover(true)}
+                onMouseLeave={() => setHover(false)}
+            >
+                <Item {...rest} state={state} />
```

Comment:
let's send the primitive state's values directly instead of the state object like:
```
<Item {...rest} isSelected={isSelected} isFocused={isFocused} isHover={isHover} />
```